source/dub/commandline.d

```diff
@@ -645,7 +645,7 @@ abstract class PackageBuildCommand : Command {
 		]);
 	}
 
-	protected void setupPackage(Dub dub, string package_name, string default_build_type = "debug")
+	protected void setupPackage(Dub dub, string package_name, Version ver = Version.unknown, string default_build_type = "debug")
```

Comment:
to avoid API breakage for people who extend the PackageBuildCommand class, can you move this new argument after default_build_type?